src/Microsoft.Net.Http.Headers/CookieHeaderParser.cs

```diff
@@ -91,7 +91,7 @@ private static int GetNextNonEmptyOrWhitespaceIndex(string input, int startIndex
 
             if (skipEmptyValues)
             {
-                while ((current < input.Length) && (input[current] == ',') && (input[current] == ';'))
+                while ((current < input.Length) && ((input[current] == ',') || (input[current] == ';')))
```

Comment:
This method is copied and customized from the generic version over here: https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.Net.Http.Headers/HeaderUtilities.cs#L188

This just looks like a typo in the customization.  We haven't started consuming this new class anywhere.